product_docs/docs/postgres_for_kubernetes/preview/rel_notes/1_28_0-rc1_rel_notes.mdx

```diff
@@ -10,6 +10,12 @@ Released: 7 November 2025
 
 This release includes the following:
 
+## Highlights
+
+We've consolidated our [private container registry](../private_edb_registries) into a single location. This change simplifies access and deployment. 
+Any customer attempting an operator upgrade, MUST configure the new unified repository pull secret! 
```

Comment:
Maybe make a note about "starting with 1.28.0". Just to highlight that you must re-configure if you want to get 1.28.0 or later.